app/objects/icons/plane-left-icon.js

```diff
@@ -0,0 +1,19 @@
+import EmberObject from '@ember/object';
+
+/**
+ * Paper plane left icon for Ember Leaflet.
+ */
+export default EmberObject.extend({
+  init() {
+    this._super(...arguments);
+    this.set('iconUrl', 'assets/images/plane-left.png');
```

Comment:
do you know if these assets need to be excluded from ember's fingerprinting in prod builds?

Reply:
I was just considering that now, and i'd say yes